wire/go.mod

```diff
@@ -5,6 +5,10 @@ go 1.17
 require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/decred/dcrd/chaincfg/chainhash v1.0.2
+	lukechampine.com/blake3 v1.1.7
```

Comment:
Nice.  I see it was just tagged last week and I hadn't checked again.  I went ahead and reviewed the changes between 1.7.1 and 1.2.1 and agree that none of them are relevant in terms of the blake3 hashing itself.  I did notice that v1.2.0 bumped the `go` directive in the `go.mod` to version 1.17 though which does bring module graph pruning and lazy loading support, so that is a nice benefit.

Updated to v1.2.1.